protocol/messenger.go

```diff
@@ -801,10 +801,14 @@ func (m *Messenger) Start() (*MessengerResponse, error) {
 	m.watchPendingCommunityRequestToJoin()
 	m.broadcastLatestUserStatus()
 	m.timeoutAutomaticStatusUpdates()
-	m.startBackupLoop()
-	err = m.startAutoMessageLoop()
-	if err != nil {
-		return nil, err
+	if !m.config.featureFlags.BackupDisabled {
```

Comment:
There is already a `settings.BackupEnabled` field that can be used for this purpose?